sandbox/Controls.qml

```diff
@@ -196,4 +196,11 @@ GridLayout {
         type: StatusBanner.Type.Danger
         statusText: "Banner"
     }
+
+    StatusProgressBar {
+        id: progressBar        
+        text: "Weak"
+        value: 0.5
+        fillColor : Theme.palette.pinColor1
+    }
```

Comment:
Shouldn't we have here `StatusPasswordStrengthIndicator` as well?

Reply:
I've created a specific page in sandbox that includes all `StatusPasswordStrengthIndicator` variants and only include the `StatusProgressBar` in Controls, as it is a base one.